apps/zui/src/app/lakes/root.tsx

```diff
@@ -27,6 +27,7 @@ export function InitLake({children}) {
   useLayoutEffect(() => {
     if (status) return
     if (lake) dispatch(updateStatus(lake.id))
+    Active.lake.sync()
   }, [lake?.id, status])
```

Comment:
Whenever the current lake ID changes, sync our info about the lake with the lake service.